packages/cicero-core/src/templatesaver.js

```diff
@@ -58,6 +59,11 @@ class TemplateSaver {
             zip.file('README.md', metadata.getREADME(), options);
         }
 
+        // save the logo.png if present
+        if (metadata.getLogo()) {
+            zip.file('logo.png', metadata.getLogo(), options);
```

Comment:
options.binary will need to be set to true:
https://stuk.github.io/jszip/documentation/api_jszip/file_data.html